.travis.yml

```diff
@@ -27,8 +27,6 @@ script:
   - 'if [ "$TRAVIS_PULL_REQUEST"  = "false" ]; then make images; fi'
   # Process 'make images' when running as a pull request
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then unset quay_repository && make images; fi'
-  # Process 'make push-images' when NOT a pull request
-  - 'if [ "$TRAVIS_PULL_REQUEST"  = "false" ]; then make push-images; fi'
```

Comment:
Also i got one doubt for this change.. I removed this to skip push images whenever we create a new tag or release branch as we are taking care this from GH actions.. But looking at Travis CI build history for each Pull requests we are building and push images with main-{version} and using that image for e2e testing https://app.travis-ci.com/github/project-codeflare/multi-cluster-app-dispatcher/builds/264401442 
I believe with this new changes it should be able to run e2e test with image build locally as we are skipping push image.